appendix-basic65.tex

```diff
@@ -5579,15 +5669,15 @@ \subsection{LEFT\$}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$C8
 \item [Format:] {\bf LEFT\$(}string{\bf,} n{\bf)}
-\item [Usage:] Returns a string
-               containing the first {\bf n} characters from the
+\item [Usage:] The first {\bf n} characters from the
```

Comment:
Prefer something similar to old text, "**A string containing** the first...
(likewise for RIGHT$)

Reply:
Fixed in both cases.